iceoryx_posh/include/iceoryx_posh/internal/popo/building_blocks/chunk_distributor.inl

```diff
@@ -131,8 +133,9 @@ inline bool ChunkDistributor<ChunkDistributorDataType>::hasStoredQueues() const
 }
 
 template <typename ChunkDistributorDataType>
-inline void ChunkDistributor<ChunkDistributorDataType>::deliverToAllStoredQueues(mepoo::SharedChunk chunk) noexcept
+inline uint64_t ChunkDistributor<ChunkDistributorDataType>::deliverToAllStoredQueues(mepoo::SharedChunk chunk) noexcept
```

Comment:
Does the publisher/server has the ability to know how many subscriber/clients they have? I am just wondering what value the return value has and what the intention behind it is.

For my point of view it would maybe make more sense to state to how many queues it was not delivered or simpler, if all queues have received the chunk or not.
But this is me just asking...

This question goes for all layers and levels. Maybe it makes more sense to have a bool which states just if this was delivered to all queues.

Reply:
I used the integer since it give more information than a boolean. We could also think about returning a `(int32_t, int32_t)` tuple with the delivered and skipped queues. If you don't mind, I would create a separate issue for this since it is not public API and for the request response the current implementation with the counter is sufficient